### ISSUE TITLE
Verify popup "X_D3DVSDT_NONE In ConvertStream"

### ISSUE THREAD

**Issue by ObiKKa** [Cxbx-Reloaded/Cxbx-Reloaded#1206]
**NOTE:** Using similar template as Patrick's other [pop-up message threads](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/issues/1157). He, another leading programmer or admin may take over this original post whenever they wish to.

**Note 2:** This test case message is actually similar to another message called _X_D3DVSDT_NORMSHORT4 In ConvertStream_. But the two different words are "NONE" here and "NORMSHORT4 there. They both might be in the same pool of code, ask the programmers.

 ---

Verify if the test case pop-up message, "_X_D3DVSDT_NONE
In ConvertStream_" is being handled correctly. That is, it's inside the vertexbuffer code sub-folder/file under the 'emud3d8' area.

If not, describe here what is wrong with it, and try to fix it.
Once it's deemed correct, disable the popup.

Test cases:
* [WWE RAW2](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/pull/1200#issuecomment-391944059)

**Comment by gandalfthewhite19890404** [comment]
Its from patrick's PR, not from master. At least its doesn't happened when I test it on master.

**Comment by RadWolfie** [comment]
@gandalfthewhite19890404, true. Let @PatrickvL decide if he want to keep a record for this test case. I think it's fine to have individual issue creation for this test case.

**Comment by ObiKKa** [comment]
Yeah, I was thinking to combine this issue inside the older issue for the NORMSHORT4 test case (Coz their sample lists are tiny for now). But some may prefer to keep them separate...

**Comment by PatrickvL** [comment]
It's a separate test case, so it wouldn't be merged with the other (similarly named) one.
